app/services/usps_in_person_proofing/date_validator.rb

```diff
@@ -0,0 +1,37 @@
+module UspsInPersonProofing
+  # Validator that can be attached to a form or other model
+  # to verify that specific supported fields are comparable to other dates.
```

Comment:
This validator is to make sure a user's age is within an accepted range right? Would it be more accurate/clear to say "...to verify that user input in supported date fields fall within an accepted range".